0.28.2: `src/validators/_tld.txt` is missing from sdist

The `src/validators/_tld.txt` file is missing from the 0.28.2 sdist and this is causing 20 tests to fail:
```
FAILED tests/test_domain.py::test_returns_true_on_valid_top_level_domain[example.com-True-False-False]
FAILED tests/test_domain.py::test_returns_true_on_valid_top_level_domain[exa_mple.com-True-False-True]
FAILED tests/test_domain.py::test_returns_true_on_valid_top_level_domain[xn----gtbspbbmkef.xn--p1ai-True-False-False]
FAILED tests/test_domain.py::test_returns_true_on_valid_top_level_domain[underscore_subdomain.example.com-True-False-True]
FAILED tests/test_domain.py::test_returns_true_on_valid_top_level_domain[someThing.versicherung.-True-True-False]
FAILED tests/test_domain.py::test_returns_true_on_valid_top_level_domain[11.com-True-False-False]
FAILED tests/test_domain.py::test_returns_true_on_valid_top_level_domain[3.cn.-True-True-False]
FAILED tests/test_domain.py::test_returns_true_on_valid_top_level_domain[_example.com-True-False-True]
FAILED tests/test_domain.py::test_returns_true_on_valid_top_level_domain[example_.com-True-False-True]
FAILED tests/test_domain.py::test_returns_true_on_valid_top_level_domain[somerandomexample.xn--fiqs8s-True-False-False]
FAILED tests/test_domain.py::test_returns_failed_validation_invalid_top_level_domain[example.266-True-False-False]
FAILED tests/test_domain.py::test_returns_failed_validation_invalid_top_level_domain[exa_mple.org_-True-False-True]
FAILED tests/test_domain.py::test_returns_failed_validation_invalid_top_level_domain[xn----gtbspbbmkef.xn-p1ai-True-False-False]
FAILED tests/test_domain.py::test_returns_failed_validation_invalid_top_level_domain[underscore_subdomain.example.flat-True-False-True]
FAILED tests/test_domain.py::test_returns_failed_validation_invalid_top_level_domain[someThing.versicherung.reddit.-True-True-False]
FAILED tests/test_domain.py::test_returns_failed_validation_invalid_top_level_domain[11.twitter-True-False-False]
FAILED tests/test_domain.py::test_returns_failed_validation_invalid_top_level_domain[3.cnx.-True-True-False]
FAILED tests/test_domain.py::test_returns_failed_validation_invalid_top_level_domain[_example.#13-True-False-True]
FAILED tests/test_domain.py::test_returns_failed_validation_invalid_top_level_domain[example_.fo-ul-True-False-True]
FAILED tests/test_domain.py::test_returns_failed_validation_invalid_top_level_domain[somerandomexample.xn-n-fiqs8s-True-False-False]
```

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -9,6 +9,22 @@ Note to self: Breaking changes must increment either
 
 -->
 
+## 0.28.3 (2024-05-25)
+
+_**Breaking**_
+
+> No breaking changes were introduced in this version.
+
+_**Features**_
+
+> No features were introduced in this version.
+
+_**Maintenance**_
+
+- hotfix: ensure `_tld.txt` is in `sdist` and `bdist` by @yozachar in [#379](https://github.com/python-validators/validators/pull/379)
+
+**Full Changelog**: [`0.28.2...0.28.3`](https://github.com/python-validators/validators/compare/0.28.2...0.28.3)
+
 ## 0.28.2 (2024-05-24)
 
 _**Breaking**_
@@ -26,6 +42,7 @@ _**Maintenance**_
 - fix(ip_address): properly handle private is false by @grleblanc in [#374](https://github.com/python-validators/validators/pull/374)
 - chore(url): allow symbols and pictographs in url by @prousso in [#375](https://github.com/python-validators/validators/pull/375)
 - build(deps): bump requests from 2.31.0 to 2.32.0 in /package by @dependabot in [#376](https://github.com/python-validators/validators/pull/376)
+- chore: fix typo; update dev deps; bump version by @yozachar in [#377](https://github.com/python-validators/validators/pull/377)
 
 **Full Changelog**: [`0.28.1...0.28.2`](https://github.com/python-validators/validators/compare/0.28.1...0.28.2)
 

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -8,7 +8,7 @@ Hi, to start, you need the following installed on your system.
 4. (Optional/Recommended) NodeJS for type checking
 5. (Optional/Recommended) [mise](https://github.com/jdx/mise) to manage multiple versions of Python & NodeJS.
 
-First [fork this repository](https://github.com/python-validators/validators/fork). Clone it to your system. Install development dependencies.
+First [fork this repository](https://github.com/python-validators/validators/fork). Uncheck "fork only `master`", because for versioned docs you'll need `gh-pages` too. Clone it to your system. Install development dependencies.
 
 ```sh
 # clone repository
@@ -78,11 +78,11 @@ $ python -m http.server -d docs/_build/web
     $ git push upstream --tag
     ```
 
-4. To update versioned docs, you must track the `gh-pages` onto a local branch. `git checkout --track upstream/gh-pages`, once.
-5. Checkout to the tag you want to include in the versioned documentation `git checkout TAG_NAME`.
-6. Then using [`mike`](https://github.com/jimporter/mike) (which is already a dev dependency) run `mike deploy -p -u VERSION stable`.
-7. Or use `mike deploy -p -u dev master`, which will deploy docs in the CURRENT commit as the `latest` documentation.
-8. Run `./package/roll.sh` (or `./package/roll.ps1`) to generate both sdist and bdist.
+4. To prevew versioned docs, run `mike serve` (`mike` is already a dev dependency).
+5. To update it, checkout to the tag you want to include in the versioned documentation `git checkout TAG_NAME`.
+6. Then run `mike deploy -p -u VERSION stable` OR run `mike deploy -p -u dev master`,
+7. Which will deploy docs in the CURRENT commit as the `latest` documentation, onto `gh-pages` branch.
+8. Run `./package/roll.sh` (or `./package/roll.ps1`) to generate both `sdist` and `bdist`.
 9. Install [`twine`](https://pypi.org/project/twine) using [`pipx`](https://pipx.pypa.io) to upload package to PyPI.
 
     ```sh

--- a/SECURITY.md
+++ b/SECURITY.md
@@ -4,7 +4,7 @@
 
 | Version    | Supported          |
 | ---------- | ------------------ |
-| `>=0.28.2` | :white_check_mark: |
+| `>=0.28.3` | :white_check_mark: |
 
 ## Reporting a Vulnerability
 

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -88,7 +88,7 @@ include = ["validators*"]
 namespaces = false
 
 [tool.setuptools.package-data]
-validators = ["py.typed"]
+validators = ["py.typed", "_tld.txt"]
 
 [tool.setuptools.dynamic]
 version = { attr = "validators.__version__" }

--- a/src/validators/__init__.py
+++ b/src/validators/__init__.py
@@ -105,4 +105,4 @@ __all__ = (
     "validator",
 )
 
-__version__ = "0.28.2"
+__version__ = "0.28.3"

--- a/src/validators/_tld.txt
+++ b/src/validators/_tld.txt
@@ -1,4 +1,4 @@
-# Version 2024040300, Last Updated Wed Apr  3 07:07:01 2024 UTC
+# Version 2024052400, Last Updated Fri May 24 07:07:01 2024 UTC
 AAA
 AARP
 ABB

--- a/src/validators/domain.py
+++ b/src/validators/domain.py
@@ -10,6 +10,7 @@ from .utils import validator
 
 def _iana_tld():
     """Load IANA TLDs as a Generator."""
+    # source: https://data.iana.org/TLD/tlds-alpha-by-domain.txt
     with Path(__file__).parent.joinpath("_tld.txt").open() as tld_f:
         _ = next(tld_f)  # ignore the first line
         for line in tld_f:

